tests/data/aws/route53.py

```diff
@@ -150,3 +150,53 @@
         ],
     ),
 ]
+
+# This data simulates a parent zone delegating authority to a subzone.
+GET_ZONES_WITH_SUBZONE = [
```

Comment:
Looks like the tests are not done. Once this are finished let me know. I've tested it manually and I think it looks good but would like to have integ test coverage.